plugins/dynatrace/README.md

```diff
@@ -51,6 +53,23 @@ metadata:
 
 The `DYNATRACE_ENTITY_ID` can be found in Dynatrace by browsing to the entity (a service, synthetic, frontend, workload, etc.). It will be located in the browser address bar in the `id` parameter and has the format `ENTITY_TYPE-ENTITY_ID`, where `ENTITY_TYPE` will be one of `SERVICE`, `SYNTHETIC_TEST`, or other, and `ENTITY_ID` will be a string of characters containing uppercase letters and numbers.
 
+##### Viewing Recent Synthetics Results
+
+To show recent results from a Synthetic Monitor, add the following annotation to `catalog-info.yaml`:
+
+```yaml
+# catalog-info.yaml
+# [...]
+metadata:
+  annotations:
+    dynatrace.com/dynatrace-synthetics-ids: SYNTHETIC_ID, SYNTHETIC_ID_2, ...
```

Comment:
This annotation should just be named `dynatrace.com/synthetics-ids` - no need to repeat the word. The other existing annotation is a bit different since it wants to clarify the "entity" word which has a specific meaning in Backstage itself.

Is it most common to have one id here, or many? Was wondering if it should be `id` instead of `ids`, and still supporting multiple as an added feature so to speak

Reply:
that's a good point, I'll change the annotation name. 
It is fairly common to have multiple Ids (at least at my company) for synthetics.